tests/unit/anchore_engine/services/policy_engine/engine/vulns/test_deduper.py

```diff
@@ -550,6 +551,28 @@ class TestImageVulnerabilitiesDeduplicator:
                 2,
                 id="multiple-nvd-refs",
             ),
+            pytest.param(
+                [
+                    VulnerabilityMatch(
+                        vulnerability=Vulnerability(
+                            feed="vulnerabilities",
+                            feed_group="nvdv2:cves",
+                            vulnerability_id="CVE-2019-12904",
+                        ),
+                        nvd=[NVDReference(vulnerability_id="CVE-2019-12904")],
+                    ),
+                    VulnerabilityMatch(
+                        vulnerability=Vulnerability(
+                            feed="vulnerabilities",
+                            feed_group="nvd:cves",
+                            vulnerability_id="CVE-2019-12904",
+                        ),
+                        nvd=[NVDReference(vulnerability_id="CVE-2019-12904")],
+                    ),
+                ],
+                1,
+                id="nvdv2-vs-nvd",
+            ),
```

Comment:
Neat! Can you add a similar tests for nvd vs ghsa and ghsa vs random identifier to mimic 3rd party sources

Reply:
Yes, wil do

Reply:
I've added those.